### PR TITLE
Hotfix: disaster

### PR DESCRIPTION
**Pull request by illright**

### CODE REVIEW

File: innopoints/config/dev.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 from innopoints.config.common import *
 
 
+# SQLALCHEMY_ECHO = True  # Will log all the issued SQL statements
```
Comment (aabounegm) [inline innopoints/config/dev.py:7]: Why is it commented out, if it is the dev config?

File: innopoints/views/account.py [hunk 2]
```diff
@@ -69,12 +68,12 @@
...
     that packs the rest of the fields into a JSON payload and returns it with the time."""
-    payload = db.func.row_to_json(as_row(subquery)).cast(JSONB) - 'entry_time'
+    # payload = db.func.row_to_json(query).cast(JSONB) - 'entry_time'
     return db.session.query(
-        'entry_time',
+        subquery.c.entry_time.label('entry_time'),
         db.literal(event_type).label('type'),
-        payload.label('payload')
-    ).select_from(subquery)
+        (db.func.row_to_json(db.literal_column(subquery.name)).cast(JSONB) - 'entry_time').label('payload')
```
Comment (aabounegm) [inline innopoints/views/account.py:75]: How did inlining `payload` fix anything?

File: innopoints/config/dev.py (hunk 1 above)
Comment (illright) [reply innopoints/config/dev.py:7]: Because it's ridiculous even in the dev config

Comment (illright) [reply innopoints/config/dev.py:7]: I just left it to know where to find it later, but after I'm done, I'll remove it entirely

File: innopoints/views/account.py (hunk 2 above)
Comment (illright) [reply innopoints/views/account.py:75]: It's not only a matter of inlining

**Pull request merged**